babel/package.json

```diff
@@ -0,0 +1,24 @@
+{
+  "name": "example-applications-babel",
```

Comment:
**Comment applies to _all_ examples**
Package name is odd. Why not `"example-babel-app"` or `"babel-example"`?